elastic4s-domain/src/main/scala/com/sksamuel/elastic4s/analysis/TokenFilter.scala

```diff
@@ -96,7 +96,7 @@ case class WordDelimiterGraphTokenFilter(
   def protectedWords(words: String): WordDelimiterGraphTokenFilter       = copy(protectedWords = words.some)
   def protectedWordsPath(path: String): WordDelimiterGraphTokenFilter    = copy(protectedWordsPath = path.some)
   def splitOnNumerics(bool: Boolean): WordDelimiterGraphTokenFilter      = copy(splitOnNumerics = bool.some)
-  def stemEnglishPossesive(bool: Boolean): WordDelimiterGraphTokenFilter = copy(stem_english_possessive = bool.some)
+  def stemEnglishPossesive(bool: Boolean): WordDelimiterGraphTokenFilter = copy(stemEnglishPossessive = bool.some)
```

Comment:
It would be neat to deprecate this method and fix the typo in a new `stemEnglishPossessive` method.